src/components/FallenHeroes.tsx

```diff
@@ -53,12 +54,12 @@ function FallenHeroes() {
         <button type="submit" className="btn input_fallen">
           Send to the coffin
         </button>
-        <header className="header">
-          <button className="btn list_fallen">List fallen heroes</button>
-          <div className="fallen__heroes__panel">
-            <ul className="fallen_heroes__list">{campaign.fallenHeroes}</ul>
-          </div>
-        </header>
+        <h3>The Fallen:</h3>
+        <ul className="fallen_heroes__list">
+          {campaign.fallenHeroes.map((fallenHero: string): JSX.Element => {
```

Comment:
Type annotation is redundant here. TypeScript can infer the type.

Reply:
yeah true. It is not incorrect specify this expressly right?

Reply:
adjusted